src/mon/commands/Command.h

```diff
@@ -0,0 +1,333 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2019 SUSE LLC <contact@suse.com>
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ *
+ */
+
+#ifndef CEPH_MON_COMMAND_H
+#define CEPH_MON_COMMAND_H
+
+#include <string>
+#include <set>
+#include <sstream>
+
+#include <boost/intrusive_ptr.hpp>
+
+#include "mon/Monitor.h"
+#include "mon/MonOpRequest.h"
+
+#include "include/types.h"
+#include "include/encoding.h"
+#include "include/ceph_assert.h"
+#include "include/Context.h"
+
+#include "common/cmdparse.h"
+#include "common/RefCountedObj.h"
+
+using namespace std;
```

Comment:
i'd suggest avoid `using namespace` in header files.

Reply:
+1

Reply:
It's probably worth briefly elaborating on the reason for this advice. In header code, this is almost always something
you want to avoid (and it's rampant in Ceph's code) because it will pollute the namespace of anything that uses the
header. This affects various kinds of lookup in ways that can range from merely a nuisiance to really hairy to untangle. 
Good discussion here: https://isocpp.org/wiki/faq/coding-standards#using-namespace-std